kpi/management/commands/migrate_json_text_to_jsonb.py

```diff
@@ -168,6 +198,19 @@ def write(*args, **kwargs):
             write('.', ending='')
         write('')
 
+        if options['drop']:
+            write('*** Dropping columns and triggers', ending='')
```

Comment:
> Dropping columns and triggers

This might stress people out. Maybe we should say `Dropping temporary migration columns and triggers` ?

Reply:
Sure :+1: